base_frame: r1_link_0
base_target_transform:
  cols: 4
  data:
  - -0.9999113245153536
  - 0.008279937905945717
  - -0.010430039991533446
  - 0.9272598032071397
  - -0.008281051020884616
  - -0.999965709490487
  - 6.353877993840021e-05
  - 0.6747845024571758
  - -0.010429156242995384
  - 0.00014990483892605072
  - 0.999945603634817
  - 0.029031133480411253
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  rows: 4
camera_gripper_transform:
  cols: 4
  data:
  - -0.999986595416234
  - 0.004171301992635167
  - 0.003067446419060371
  - -0.011743126532827816
  - -0.0041753617155257505
  - -0.9999904142063175
  - -0.0013182753965454708
  - 0.043808137769994655
  - 0.003061918090363741
  - -0.0013310654239547248
  - 0.9999944264456903
  - 0.07779778860295877
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  rows: 4
camera_matrix:
  cols: 3
  data:
  - 641.7655550407592
  - 0.0
  - 643.047388829349
  - 0.0
  - 641.7655550407592
  - 377.2470626983552
  - 0.0
  - 0.0
  - 1.0
  rows: 3
camera_name: camera
distortion_coefficients:
  cols: 5
  data:
  - -0.058651324415246404
  - 0.06654372516292956
  - 0.0017373645856013776
  - -0.0009708349563279144
  - -0.022730273314639577
  rows: 1
distortion_model: plumb_bob
end_effector_frame: r1_link_7
image_height: 720
image_width: 1280
projection_matrix:
  cols: 4
  data:
  - 641.7655550407592
  - 0.0
  - 643.047388829349
  - 0.0
  - 0.0
  - 641.7655550407592
  - 377.2470626983552
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  rows: 3
rectification_matrix:
  cols: 3
  data:
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  rows: 3
